protocol/0085-RVST-rewards_vesting.md

```diff
@@ -78,4 +78,29 @@ Must expose the following:
 
 ## Acceptance Criteria
 
-Too be added
+### Network parameters
+
+1. When `rewards.vesting.baseTransfer` is updated, the new value should be applied to rewards vesting at the end of the current epoch. (<a name="0085-RVST-001" href="#0085-RVST-001">0085-RVST-001</a>)
+1. When `rewards.vesting.minimumTransfer` is updated, the new value should be applied to rewards vesting at the end of the current epoch. (<a name="0085-RVST-002" href="#0085-RVST-002">0085-RVST-002</a>)
+1. When `rewards.vesting.rewardPayoutTiers` is updated, the new value should be applied when distributing rewards at the end of the current epoch. (<a name="0085-RVST-003" href="#0085-RVST-003">0085-RVST-003</a>)
+
+### Vesting / vested accounts
+
+1. A party should have one vesting account per asset. Rewards distributed from reward pools should be transferred to the correct vesting account. (<a name="0085-RVST-004" href="#0085-RVST-004">0085-RVST-004</a>)
+1. A party should have one vested account per asset. Rewards distributed from vesting accounts should be transferred to the correct vested account. (<a name="0085-RVST-005" href="#0085-RVST-005">0085-RVST-005</a>)
+1. Funds **cannot** be transferred from a vesting account by a user. (<a name="0085-RVST-006" href="#0085-RVST-006">0085-RVST-006</a>)
+1. Funds **can** be transferred from a vested account by a user. (<a name="0085-RVST-007" href="#0085-RVST-007">0085-RVST-007</a>)
+1. Funds **cannot** be transferred to a vested account by a user. (<a name="0085-RVST-008" href="#0085-RVST-008">0085-RVST-008</a>)
+
+### Vesting mechanics
+
+1. If a party has unlocked rewards in a vesting account (expressed in quantum) strictly greater than the network parameter `rewards.vesting.minimumTransfer`. Rewards should be transferred to the respective vested account for the asset at the end of the epoch as per the formula defined in the specification. (<a name="0085-RVST-009" href="#0085-RVST-009">0085-RVST-009</a>)
```

Comment:
(sorry, nitpick warning)

Since we're starting with an 'If', could we connect the sentence with a 'then'? Or somehow improve the flow.